Makefile.am

```diff
@@ -0,0 +1,304 @@
+desktopdir = $(datarootdir)/applications
+desktop_DATA = dpkgfiles/armory.desktop dpkgfiles/armoryoffline.desktop \
+			  dpkgfiles/armorytestnet.desktop
+CLEANFILES = $(desktop_DATA)
+
+PYTHON_INCLUDES=@PYTHON_INCLUDES@
+
+LINK = $(CXX)
+
+LDLIBS = -lpthread -Lmdb
+CPPFLAGS = $(ARMORY_CPPFLAGS) -IcppForSwig/cryptopp -IcppForSwig/mdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
+CXXCPP = $(CPPFLAGS) $(PYTHON_INCLUDES)
+
+if TARGET_DARWIN
+LDFLAGS += -undefined dynamic_lookup -headerpad_max_install_names
+LDLIBS += -lc++
+CXXCPP += -stdlib=libc++
+endif
+
+if TARGET_LINUX
+LDLIBS += -lrt
```

Comment:
-lrt was being added to LIBS by the check in the configure script, but that
was it LIBS was used for, so when I switched to hard coded -lrt here, I
just removed the useless $(LIBS).
On Sep 1, 2015 10:27 AM, "Douglas Roark" <notifications@github.com> wrote:

> In Makefile.am
> <https://github.com/etotheipi/BitcoinArmory/pull/304#discussion_r38424326>
> :
>
> > +PYTHON_INCLUDES=@PYTHON_INCLUDES@
> > +
> > +LINK = $(CXX)
> > +
> > +LDLIBS = -lpthread -Lmdb
> > +CPPFLAGS = $(ARMORY_CPPFLAGS) -IcppForSwig/cryptopp -IcppForSwig/mdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
> > +CXXCPP = $(CPPFLAGS) $(PYTHON_INCLUDES)
> > +
> > +if TARGET_DARWIN
> > +LDFLAGS += -undefined dynamic_lookup -headerpad_max_install_names
> > +LDLIBS += -lc++
> > +CXXCPP += -stdlib=libc++
> > +endif
> > +
> > +if TARGET_LINUX
> > +LDLIBS += -lrt
>
> Why was $(LIBS) removed? Was it a typo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/etotheipi/BitcoinArmory/pull/304/files#r38424326>.
>